test/cypress.json

```diff
@@ -0,0 +1,6 @@
+{
+    "baseUrl": "http://localhost:6060/#",
```

Comment:
Can we use the test folder instead, and inject the access token? This way we can use a non-hardcoded access token which could have additional scopes such as those needed for creating comments.

Reply:
The example files accept a token via the `TOKEN` environment variable.